src/models/units.ts

```diff
@@ -1,5 +1,10 @@
 import { types } from "mobx-state-tree";
 import { Unit } from "../authoring";
+import { LegendItem } from "../components/spaces/breeding/breeding-data";
+import { PieChartData } from "../components/charts/pie-chart";
+// @ts-ignore
+import * as colors from "../components/colors.scss";
```

Comment:
Just a note that we ended up using this method in CLUE and the AP for importing Sass variables:
https://mattferderer.com/use-sass-variables-in-typescript-and-javascript

I'm not saying that anything needs to change (especially since there are a handful of these imports in Cbio) - just mentioning it.